rules/apple_patched.bzl

```diff
@@ -80,12 +91,40 @@ def _apple_framework_import_modulemap_impl(ctx):
         ],
     )
 
+    modulemap_list = old_objc_provider.module_map.to_list()
+    hdrs_list = old_cc_info.compilation_context.headers.to_list()
+    hdrs = [h.path for h in hdrs_list]
+
+    imported_framework_name = _find_imported_framework_name(hdrs)
+    vfs = make_vfsoverlay(
+        ctx,
+        hdrs = hdrs_list,
+        module_map = modulemap_list,
+        # Consider aligning this code with the way that frameworks are
+        # imported from rules/library.bzl. There are effectivly 2 different ways
+        # that frameworks are being imported.  Otherwise, a test case for
+        # swiftmodules ensuring it extracts from the provider. The swiftmodules
+        # for this case should be loaded into the VFS
+        swiftmodules = [],
+        private_hdrs = [],
+        has_swift = False,
+        framework_name = imported_framework_name,
+    )
+    framework_info = FrameworkInfo(
```

Comment:
Perhaps this might need to be conditional until `virtualize_frameworks` is default.